azimuthController:
    iMax: 0
    iZone: 0
    kD: 0
    kF: 0
    kI: 0
    kP: 10
    maxAcceleration: 1080
    maxSpeed: 1440
    tolerance: 0

locationX: 0.50625
locationY: 0.50625
wheelDiameter: 0.2875

motors:
    0:
        inverted: false
        maxSpeed: 101
    1:
        inverted: false
        maxSpeed: 101
sensor:
    inverted: false
    offset: -43.07
    ratio: 0.0027778

wheelController:
    iMax: 0
    iZone: 1.5
    kD: 0
    kF: 0
    kI: 0
    kP: 0.5
    tolerance: 0
